lib/main.dart

```diff
@@ -1,32 +1,67 @@
 import 'package:flutter/material.dart';
+import '../../../utils/UIConstants.dart';
+import 'custom_widgets.dart';
+import '../../../models/movie.dart';
 
 void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
-  messagePrint() {
-    print('Function Called');
-  }
-
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
-        home: Scaffold(
-            body: SafeArea(
-                top: true,
-                bottom: true,
-                left: true,
-                right: true,
-                child: Center(
-                  child: Text(
-                    'Example of SafeArea Widget in Flutter',
-                    style: TextStyle(
-                      fontSize: 26,
-                      color: Colors.black,
-                    ),
-                    textAlign: TextAlign.center,
+    Movie movie = Movie.fromStatic();
+    return MaterialApp(
+      home: Scaffold(
+        backgroundColor: UIConstants.detailsBackgroundColor,
+        body: SafeArea(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              AspectRatio(
+                aspectRatio: UIConstants.backdropAspectRatio,
+                child: DetailsBackdropSection(
+                  pathToBackdropImg: movie.pathToBackdropImg,
+                  genres: movie.genres,
+                  decoration: buildBackdropDecoration(
+                    Alignment.bottomCenter,
+                    Alignment.topCenter,
                   ),
-                ))));
+                ),
+              ),
+              DetailsTitleSection(
+                movieTitle: movie.title,
+                movieReleaseDate: movie.releaseDate,
+                decoration: buildBackdropDecoration(
+                  Alignment.topCenter,
+                  Alignment.bottomCenter,
+                ),
+              ),
+              DetailsPosterSection(
+                pathToPosterImg: movie.pathToPosterImg,
+                movieScore: movie.score,
+              ),
+              MovieOverviewText(
+                movieOverviewTxt: movie.overview,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  BoxDecoration buildBackdropDecoration(
```

Comment:
the build method should be the last on the file, raise this one above it

Reply:
fixed